bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/BusEnergyMeter.xml

```diff
@@ -31,6 +33,12 @@
 				<description>A "5" must be added before the energy meter address: 5N with N=[1-255]. Example energy meter "42" -->
 					where="542"</description>
 			</parameter>
+
+			<parameter name="energyRefreshPeriod" type="integer" min="30" max="1440" required="true">
```

Comment:
This new parameter should be mentioned in the documentation.

Reply:
ok